clangd-wrapper/main.cpp

```diff
@@ -0,0 +1,151 @@
+#include "bcc/version.h"
+#include <algorithm>
+#include <filesystem>
+#include <iostream>
+#include <optional>
+#include <string>
+#include <vector>
+#include <boost/process.hpp>
+#include <boost/program_options.hpp>
+
+#ifdef _WIN32
```

Comment:
```suggestion
#ifndef _WIN32
```

shouldn't these header only be included in case we build _not_ for Windows? #53 is still missing :)

Reply:
whoops! I'm surprised it built like this